ci/create.workflow

```diff
@@ -0,0 +1,11 @@
+workflow "Create Zenodo Depostion" {
+  resolves = "create"
+}
+action "create" {
+  uses = "popperized/zenodo/create@master"
```

Comment:
in the case of this workflow we use on travis, this would need to point to the code under test rather than to the github repo. So this would become `./create`